<ui version="4.0" >
 <class>CannyDialog</class>
 <widget class="QDialog" name="CannyDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>242</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>242</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Select thresholds</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>222</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>preview</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxBlur" >
        <property name="text" >
         <string>blur</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>5</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text" >
         <string>Hi</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>spinBoxHi</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxHi" >
        <property name="maximum" >
         <number>255</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Low</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>spinBoxLow</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxLow" >
        <property name="maximum" >
         <number>255</number>
        </property>
        <property name="value" >
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>131</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="PreviewLabel" name="previewLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursor>9</cursor>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="textFormat" >
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents" >
    <bool>false</bool>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags" >
    <enum>Qt::NoTextInteraction</enum>
   </property>
   <property name="buddy" >
    <cstring>checkBoxBlur</cstring>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewLabel</class>
   <extends>QLabel</extends>
   <header>src/previewlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CannyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CannyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
